### ISSUE TITLE
Some missing sensors

### ISSUE THREAD

**Issue by Notrial** [briis/smartweather#49]
Hi, it's me again :) Been using your component for a week or so...all working nice for the most part.
Don't know if you are aware but the sensor list that component creates is a lot different then in your documentation.
I have a Tempest btw.

This is the sensor list I get:

sensor.smartweather_battery_st_xxx
sensor.smartweather_brightness
sensor.smartweather_dewpoint
sensor.smartweather_feels_like
sensor.smartweather_heat_index
sensor.smartweather_humidity
sensor.smartweather_lightning_count
sensor.smartweather_lightning_distance
sensor.smartweather_lightning_last_3hrs
sensor.smartweather_lightning_time
sensor.smartweather_pressure
sensor.smartweather_rain_last_hour
sensor.smartweather_rain_minutes_today
sensor.smartweather_rain_minutes_yesterday
sensor.smartweather_rain_rate
sensor.smartweather_rain_today
sensor.smartweather_rain_yesterday
sensor.smartweather_solar_radiation
sensor.smartweather_temperature
sensor.smartweather_uv
sensor.smartweather_wind_bearing
sensor.smartweather_wind_chill
sensor.smartweather_wind_direction
sensor.smartweather_wind_gust
sensor.smartweather_wind_speed

So for example I don't have:

wind_avg
air_density
lightning_strike_last_time - or named differently
lightning_strike_last_distance - or named differently
precip_accum_last_1hr
timestamp

There is also another issue, that is a bit bigger for me. The sensor.smartweather_pressure shows local Station Pressure, but in the tempest app it actually shows a graph of "Sea level pressure". Would it be possible to add that one as well (I get weird readings with station pressure)?

Thanks so much!

**Comment by briis** [comment]
Yes it might be time to review the docs again.
wind_speed = wind_avg as there is no direct wind_speed value in the API. (So I just named wind_avg as wind_speed) The other I might need to check. Does the Tempest have a Lightning detector?

And yes, I can add the Sea Level Pressure. Give me a few days, as I am currently working on another project.

**Comment by Notrial** [comment]
Makes sense regarding wind speed (wind average and gust are only two that are valuable, to me at least).
Regarding lightning, yep it has it. All 4 that are added by default work as intended I believe. 

Thanks again!

**Comment by briis** [comment]
I am working on the new Sea Level Pressure, but checked the code and here is the status of the sensors you think are missing:
* wind_avg: Is called **wind_speed**
* air_density: Yes, I pull the data, but I forgot to add the sensor. Will be fixed
* lightning_strike_last_time: Is called **lightning_time**
* lightning_strike_last_distance: Is called **lightning_distance**
* precip_accum_last_1hr: Is called **rain_last_hour**
* timestamp: Is not displayed, but only used in calculations. Will be removed from the Docs.

So in short, I will update the component, and the missing air_density, plus the Sea Level pressure.

**Comment by briis** [comment]
I now have something for you to test:
If you pull the below files from [this repository](https://github.com/briis/smartweather/tree/master/custom_components/smartweather) and overwrite the files in your HA custom_components directory, and then restart HA, you should get 2 new sensors *air_density* and *sea_level_pressure*. Let me know how that goes.

Files to pull:
* `sensor.py`
*  `manifest.json`

**Comment by Notrial** [comment]
Done!

- I can see the air density sensor - showing correct value
- I can also see the sea level pressure

Awesome!
Please take a look at the screenshot below. Can the pressure state be imported as sensor by any chance? I believe we have all the other sensors sorted out now (this list might help you check against the Tempest source). I would probably rename most of them the same way as Weatherflow did, to avoid the confusion. 

![image](https://user-images.githubusercontent.com/36201754/103869239-57b49380-50ca-11eb-84a2-3362a3b1bbad.png)

Also, might be better to show the lightning_strike_last_time as "last changed" (for example 48 mins ago) instead of a timestamp (but I know we can create a custom sensor and template to fix that).

Let me know if I can be of further help.

Cheers!

**Comment by briis** [comment]
Thanks for the quick feedback. If you copy the same files again, then you should now see the following new sensors:

* `ADDED` New sensor called `sensor.smartweather_air_density` showing the recorded Air Density at the station.
* `ADDED` New sensor called `sensor.smartweather_sea_level_pressure` showing the Sea Level Pressure as opposed to the Station Pressure.
* `ADDED` New sensor called `sensor.smartweather_pressure_trend` showing the Pressure trend in text.
* `ADDED` New sensor called `sensor.smartweather_lightning_last_1hrs` showing the number of Lightnings strikes for the last 1 hour

I agree that it would be a good idea to rename the sensors to reflect the Weatherflow App, but that would introduce breaking changes for current users, and I am not going to do that now. But maybe at a later state.

If this works for you, I will make an official release.

**Comment by Notrial** [comment]
All works as intended! :)
I get the breaking changes decision...makes sense.

Thanks again!

**Comment by briis** [comment]
Just released 2.2, so I will close this now. Thanks for helping out.
